--- a/Makefile
+++ b/Makefile
@@ -71,7 +71,7 @@ AWK ?= gawk
 GREP ?= egrep
 INSTALL ?= install
 M4 ?= m4 -E -E
-PYTHON ?= python -t -t -E -W error
+PYTHON ?= python3 -E -W error
 SED ?= sed
 SORT ?= LC_ALL=C sort
 UMASK ?= umask
--- a/support/Makefile.devel
+++ b/support/Makefile.devel
@@ -5,7 +5,7 @@ INSTALL ?= install
 M4 ?= m4
 SED ?= sed
 EINFO ?= echo
-PYTHON ?= python
+PYTHON ?= python3
 CUT ?= cut
 
 NAME ?= $(strip $(shell $(AWK) -F= '/^SELINUXTYPE/{ print $$2 }' /etc/selinux/config))
